script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -725,7 +725,7 @@ It's certainly something that the planet itself never thought of."
 [sha:6433ad2278c74a017e17800e9d66f5cecafb6092]{
 -- Page 347, Offset 25133.
 -- 真祖である吸血種―――わたしの場合は、そんな形而上の二次的利益を満たしてくれる最上級の食べ物が血液なの。
-For True Ancestors like myself, blood is the highest grade food that can provide such secondary metaphysical benefits.
+For bloodsuckers of the True Ancestors type like myself, blood is the highest grade of food that can provide such secondary metaphysical benefits.
```

Comment:
It does make the line a little messier.

In Japanese perhaps even more so. ( 真祖である吸血種―――わたしの場合は)

Actually a more accurate way of rending it would be: "For bloodsuckers of the True Ancestors type―such as in my case―..."

In some ways this is more of a philosophical question then: Sure these lines may be messy (metaphysical WHAT?), but they're not messy in a way that is uniquely because of its a JP>EN translation. So do you edit Nasu, or do you edit the translation, at that point?